source/docs/03-metadata.xml

```diff
@@ -1625,6 +1625,43 @@
                               <specDesc key="handList"/>
                            </specList>
                         </p>
+                        <p>
+                           <specList>
+                              <specDesc key="addDesc"/>
+                           </specList>
+                        </p>
+                        <p>In manuscripts there can be various kinds of entries, additions, corrections, marginalia and revisions; all these interventions in the “original” manuscript can be documented under <gi scheme="MEI">addDesc</gi>. However, it is important to understand that these are not additions to the musical text directly. These additions to the manuscript can come from the composer himself, copyists, typesetters, publishers, previous owners or librarians: In accordance with standard scholarly practice, these entries are usually indicated in the continuous text with an indication of the location within the manuscript as well as the means of writing used. E.g. “Auf fol. 109v links mit Bleistift von Schindler „NB (Sch.) Hier fehlen 8 Takte (auch im Chor). jedoch die eine spätere […]“” (see <ref target="https://digital.staatsbibliothek-berlin.de/werkansicht?PPN=PPN756658373&amp;PHYSID=PHYS_0398&amp;DMDID=DMDLOG_0005">facsimile</ref>).
+                        </p>
+                        <p>This entry could be encoded as follows:</p>
+                        <p>
+                           <figure>
+                              <head/>
+                              <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample089.txt" parse="text"/></egXML>
+                           </figure>
+                        </p>
+                        <p>A slightly more structured form would be:</p>
+                        <p>
+                           <figure>
+                              <head/>
+                              <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample090.txt" parse="text"/></egXML>
+                           </figure>
+                        </p>
+                        <p>However, although a standardization of these descriptions is difficult to realize due to different practices, these transcriptions – as in the musical text – can be marked by means of <gi scheme="MEI">add</gi>, <gi scheme="MEI">del</gi>, (see module MEI.edittrans) etc. (see <ptr target="#editTrans"/>) and assigned to a specific scribe by <att>hand</att> (see <gi scheme="MEI">handList</gi>):</p>
```

Comment:
I wouldn't say this is "difficult." What you're trying to say here is that you can mark up these additions with some other things, so just say this directly; you don't need to make excuses for the contents of a manuscript, you just need to alert users to the fact that they can add additional context to their markup.

```suggestion
                        <p>These transcriptions – as in the musical text – can also be marked by means of <gi scheme="MEI">add</gi>, <gi scheme="MEI">del</gi>, (see module MEI.edittrans) etc. (see <ptr target="#editTrans"/>) and assigned to a specific scribe by <att>hand</att> (see <gi scheme="MEI">handList</gi>):</p>
```

Reply:
@gucl-mu I would like to accept this suggestion. Are you also fine with it?